<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="2" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-08T11:12:28 IST" name="Suite" finished-at="2021-09-08T11:13:17 IST" duration-ms="48944">
    <groups>
    </groups>
    <test started-at="2021-09-08T11:12:28 IST" name="Test" finished-at="2021-09-08T11:13:17 IST" duration-ms="48944">
      <class name="testcase.DeleteLead">
        <test-method signature="runDelete(java.lang.String)[pri:0, instance:testcase.DeleteLead@77847718]" started-at="2021-09-08T11:12:28 IST" name="runDelete" finished-at="2021-09-08T11:12:28 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testcase.DeleteLead.runDelete(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testcase.DeleteLead.runDelete(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runDelete -->
      </class> <!-- testcase.DeleteLead -->
      <class name="testcase.EditLead">
        <test-method signature="runEdit(java.lang.String,java.lang.String)[pri:0, instance:testcase.EditLead@5ffead27]" started-at="2021-09-08T11:12:28 IST" name="runEdit" finished-at="2021-09-08T11:12:28 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testcase.EditLead.runEdit(java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testcase.EditLead.runEdit(java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runEdit -->
      </class> <!-- testcase.EditLead -->
      <class name="testcase.CreateLead">
        <test-method is-config="true" signature="PreDeclaration()[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:12:28 IST" name="PreDeclaration" finished-at="2021-09-08T11:12:46 IST" duration-ms="17448" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreDeclaration -->
        <test-method signature="runCreate(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:12:46 IST" name="runCreate" data-provider="sentData" finished-at="2021-09-08T11:12:51 IST" duration-ms="4813" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[GK Tech]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gokulraj]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[S]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[99]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runCreate -->
        <test-method is-config="true" signature="PostDeclaration()[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:12:51 IST" name="PostDeclaration" finished-at="2021-09-08T11:12:55 IST" duration-ms="4176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostDeclaration -->
        <test-method is-config="true" signature="PreDeclaration()[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:12:55 IST" name="PreDeclaration" finished-at="2021-09-08T11:13:09 IST" duration-ms="13738" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreDeclaration -->
        <test-method signature="runCreate(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:13:09 IST" name="runCreate" data-provider="sentData" finished-at="2021-09-08T11:13:13 IST" duration-ms="4392" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[GS Tech]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Raja]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[S]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[98]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runCreate -->
        <test-method is-config="true" signature="PostDeclaration()[pri:0, instance:testcase.CreateLead@7f3b84b8]" started-at="2021-09-08T11:13:13 IST" name="PostDeclaration" finished-at="2021-09-08T11:13:17 IST" duration-ms="4190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostDeclaration -->
      </class> <!-- testcase.CreateLead -->
      <class name="testcase.MergeLead">
        <test-method signature="runCreate(java.lang.String)[pri:0, instance:testcase.MergeLead@57a3af25]" started-at="2021-09-08T11:13:17 IST" name="runCreate" finished-at="2021-09-08T11:13:17 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testcase.MergeLead.runCreate(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testcase.MergeLead.runCreate(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runCreate -->
      </class> <!-- testcase.MergeLead -->
      <class name="testcase.DuplicateLead">
        <test-method signature="runDuplicate(java.lang.String)[pri:0, instance:testcase.DuplicateLead@2b662a77]" started-at="2021-09-08T11:13:17 IST" name="runDuplicate" finished-at="2021-09-08T11:13:17 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testcase.DuplicateLead.runDuplicate(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testcase.DuplicateLead.runDuplicate(java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : sentData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runDuplicate -->
      </class> <!-- testcase.DuplicateLead -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
